x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -226,10 +170,79 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
               </EuiFlexItem>
             )}
+          </EuiFlexGroup>
+
+          <EuiSpacer size="m" />
+
+          <EuiFlexGroup direction="row" gutterSize="m" responsive={false} alignItems="baseline">
+            {viewBySwimlaneOptions.length > 0 && (
+              <>
+                <EuiFlexItem grow={false}>
+                  <EuiSelect
+                    prepend={i18n.translate('xpack.ml.explorer.viewByLabel', {
+                      defaultMessage: 'View by',
+                    })}
+                    compressed
+                    id="selectViewBy"
+                    options={mapSwimlaneOptionsToEuiOptions(viewBySwimlaneOptions)}
+                    value={viewBySwimlaneFieldName}
+                    onChange={(e) => explorerService.setViewBySwimlaneFieldName(e.target.value)}
+                  />
+                </EuiFlexItem>
+              </>
+            )}
+
+            <EuiFlexItem grow={true}>
+              <SeverityControl
+                value={swimLaneSeverity ?? 0}
+                onChange={useCallback((update) => {
+                  explorerService.setSwimLaneSeverity(update);
+                }, [])}
+              />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+
+          <EuiSpacer size="m" />
 
+          <EuiFlexGroup direction="row" gutterSize="m" responsive={false} alignItems="center">
             <EuiFlexItem grow={false}>
-              <AnomalyTimelineHelpPopover />
+              <div className="panel-sub-title">
+                {viewByLoadedForTimeFormatted && (
+                  <FormattedMessage
+                    id="xpack.ml.explorer.sortedByMaxAnomalyScoreForTimeFormattedLabel"
+                    defaultMessage="(Sorted by max anomaly score for {viewByLoadedForTimeFormatted})"
+                    values={{ viewByLoadedForTimeFormatted }}
+                  />
+                )}
+                {isDefined(viewByLoadedForTimeFormatted) ? null : (
+                  <FormattedMessage
+                    id="xpack.ml.explorer.sortedByMaxAnomalyScoreLabel"
+                    defaultMessage="(Sorted by max anomaly score)"
+                  />
+                )}
+                {filterActive === true && viewBySwimlaneFieldName === VIEW_BY_JOB_LABEL && (
+                  <FormattedMessage
+                    id="xpack.ml.explorer.jobScoreAcrossAllInfluencersLabel"
+                    defaultMessage="(Job score across all influencers)"
+                  />
+                )}
+              </div>
             </EuiFlexItem>
+
+            {selectedCells ? (
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmpty
+                  size="xs"
+                  onClick={setSelectedCells.bind(null, undefined)}
+                  data-test-subj="mlAnomalyTimelineClearSelection"
+                >
+                  <FormattedMessage
+                    id="xpack.ml.explorer.clearSelectionLabel"
+                    defaultMessage="Clear"
```

Comment:
Thinking `Clear selection` is actually better here, especially as there is plenty of width available.